server/controllers/articles.js

```diff
@@ -0,0 +1,103 @@
+import { validationResponse } from '@helpers/validationResponse';
+import Response from '@helpers/Response';
+import models from '@models';
+import validateRating from '@validations/rating';
+
+const { User, Article, Rating } = models;
+
+/**
+ * @exports ArticleController
+ * @class ArticleController
+ * @description Handles Article related actions
+ * */
+class ArticleController {
+  /**
+   * Rate an article
+   * @async
+   * @param  {object} req - Request object
+   * @param {object} res - Response object
+   * @param {object} next The next middleware
+   * @return {json} Returns json object
+   * @static
+   */
+  static async rate(req, res, next) {
+    const userId = req.decoded.id;
+
+    try {
+      // Validate the rating
+      const articleDetails = await validateRating(req.body);
+
+      const { articleId, rate } = articleDetails;
```

Comment:
Nice job! The article-id here seems to be provided by the user. Why not include it as a param to save the user from finding the article-id before rating.